client/src/components/Leaderboard.jsx

```diff
@@ -92,14 +92,14 @@ class Leaderboard extends React.Component {
     return (
       <div className="leaderboard-container" id="leaderboard-container">
         <Friend />
-        <div id="list">
+        <div id="leaderboard-list">
           <div id="list-header">
             <form>
-              <input type="range" id="friend-slider" name="friend-slider" min="0" max="1" value={this.state.friendsMode === 'leaders'
+              Leaderboard&nbsp;&nbsp;<input type="range" id="friend-slider" name="friend-slider" min="0" max="1" value={this.state.friendsMode === 'leaders'
```

Comment:
Had no idea what the nbsp was, but looked it up. Really cool!

Reply:
Yeah it's a good one. Comes in handy. (Not mine either.)